install.sh

```diff
@@ -29,6 +29,11 @@ cat $HERE/nginx-site | sed "s|DIR|$MZ_LOCATION|" | sudo tee /etc/nginx/sites-ava
 sudo ln -sf /etc/nginx/sites-available/musicazoo /etc/nginx/sites-enabled/musicazoo
 
 
+echo "=> Disabling unwanted programs"
+killall -q xscreensaver
+echo -n > $HOME/.config/lxsession/LXDE/autostart
```

Comment:
This is a bit broad -- I'd rather not just arbitrarily delete the entire contents of a configuration file that isn't xscreensaver-specific.

Reply:
My rationale is that we really don't need much of a desktop environment at all as long as VLC can enter full screen, so none of LXDE's autostart programs are needed.

Reply:
It's possible that someone might install this on a system that they also use for something else, so I still don't like blowing away an entire configuration file.